helm_deploy/prisoner-offender-search/snapshot.sh

```diff
@@ -16,35 +16,35 @@ http_es() {
 }
 
 # Register snapshot repo if not already
-if ! http_es GET $API_ENDPOINT/_snapshot/$NAMESPACE &>/dev/null; then
+if ! http_es GET "$API_ENDPOINT/_snapshot/$NAMESPACE" &>/dev/null; then
   # If the snapshot repo with same name as namespace does not exist, create it.
   settings="{
     \"bucket\": \"${BUCKET_NAME}\",
     \"region\": \"eu-west-2\",
     \"role_arn\": \"${SNAPSHOT_ROLE_ARN}\",
     \"base_path\": \"${NAMESPACE}\"
   }"
-  http_es --print=Hbh POST $API_ENDPOINT/_snapshot/$NAMESPACE \
+  http_es --print=Hbh POST "$API_ENDPOINT/_snapshot/$NAMESPACE" \
     type="s3" \
     settings:="${settings}"
 fi
 
 # Delete latest snapshot if exists
-echo -e "\nDeleting existing snapshot"
```

Comment:
moved the echo into the `while` block so that only echos the delete if we have actually deleted the previous one